src/saving/SaveManagerGUI.cs

```diff
@@ -218,11 +230,12 @@ private void DeleteOldButtonPressed()
     {
         int autoSavesToDeleteCount = (currentAutoSaveCount - 1).Clamp(0, Settings.Instance.MaxAutoSaves);
         int quickSavesToDeleteCount = (currentQuickSaveCount - 1).Clamp(0, Settings.Instance.MaxQuickSaves);
+        int oldBackupsToDeleteCount = currentBackupCount.Clamp(0, Settings.Instance.MaxQuickSaves);
```

Comment:
Why is this using MaxQuickSaves here? All backups should be deleted when cleaning out old things. (or a separate variable added for that, but that would be quite a bit of work for no real benefit).

Reply:
I don't really know to be honest, I saw it in the auto save and quick save and because this is just the numbers that are being displayed it does no harm so I just copied it over. I don't really see why there is a Max for auto or quick saves either to be honest.

Reply:
These are the numbers we are showing the player about what they are going to true. Not showing the actual count of things to delete is a major issue.

Reply:
So we should instead just not be clamping it and set the value to zero if the value is less than zero otherwise leave it untouched.

Reply:
I'd just use the current backup count as is. It can't be below zero anyway? I probably used Clamp above as I subtracted 1 so it could end up negative in those cases.

Reply:
The count will below zero in cases where there are 0 auto saves to delete and we are subtracting 1 which accounts for not deleting the most recent auto save.

Reply:
Yes? But how does that affect the count of backups? That's the line we are talking about here. I'm confused about what you are confused about...

Reply:
> that there will always be a number of auto saves that is less than or equal to this max clamp 

That is an unsafe assumption. The player can crank up the numbers, make a bunch of saves, and then lower them a lot. Meaning that the actual save count is unrelated to the setting.

Reply:
if thats the case we should clamp the number of auto saves deleted or something because that would mean that it unsynced with the actual number of saves deleted because we will __always__ delete all but the most recent save

Reply:
I suppose the code should actually be something like `Math.Max(currentAutoSaveCount - 1, 0)` and the same for currentQuickSaveCount. But doing that change needs to also verify that the actual deletion part doesn't have a bug, that's all I can say without having to read the code and understand it again.